python/nano/setup.py

```diff
@@ -73,7 +73,9 @@ def download_libs(url: str):
 
 def setup_package():
 
-    tensorflow_requires = ["intel-tensorflow==2.7.0; platform_machine=='x86_64'",
+    tensorflow_requires = ["intel-tensorflow==2.7.0; platform_machine=='x86_64' and \
+                           platform_system!='Darwin'",
+                           "tensorflow==2.7.0;platform_system=='Darwin'",
```

Comment:
this might not work for non-x86 machine such as M-series chip, you may add
```python
platform_machine=='x86_64' and platform_system=='Darwin'"
```

Reply:
> this might not work for non-x86 machine such as M-series chip, you may add
> 
> ```python
> platform_machine=='x86_64' and platform_system=='Darwin'"
> ```

done